cli/bench_command.go

```diff
@@ -66,305 +62,540 @@ type benchCmd struct {
 	deDuplicationWindow  time.Duration
 	retries              int
 	retriesUsed          bool
+	ack                  bool
 }
 
 const (
-	DefaultDurableConsumerName string = "natscli-bench"
+	DefaultDurableConsumerName string = "nats-bench"
```

Comment:
be good to prefix all of these with Bench and then not export them

Reply:
It's a durable consumer (except for ordered of course) but it's deleted at the end of a successful bench run, and also has a pretty short idle timeout, so it only really exists while the bench is running.
So you would rather the name was 'bench-natscli' ?

Reply:
I mean the variable names :) since these clutter up the the entire cli package it would be good to have them be like `benchDefaultDurableConsumerName`

Reply:
Ah ok, got ya, done.